src/Model/Rating.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace sodoff.Model;
+
+public class Rating {
+    [Key]
+    public int Id { get; set; }
+
+    public int VikingId { get; set; }
+
+    public int RankId { get; set; }
+
+    public int CategoryID { get; set; }
+
+    public int? RatedEntityID { get; set; }
+    public string? RatedUserID { get; set; }
+
+    public int Value { get; set; }
+
+    public DateTime Date { get; set; }
+
+
+    public virtual Viking? Viking { get; set; }
+
+    public virtual RatingRank? Rank { get; set; }
```

Comment:
this shouldn't allow null ... so `public virtual RatingRank Rank { get; set; }` ... probably the same with `Viking` ...